spec/integration/compare_against_solrmarc_docs_spec.rb

```diff
@@ -2,64 +2,31 @@
 
 require 'spec_helper'
 
-describe 'comparing against a well-known location full of documents generated by solrmarc' do
+RSpec.describe 'comparing against a well-known location full of documents' do
   let(:indexer) do
     Traject::Indexer.new.tap do |i|
       i.load_config_file('./lib/traject/config/folio_config.rb')
     end
   end
 
-  let(:record) { MARC::XMLReader.new(StringIO.new(marcxml)).to_a.first }
-  let(:copy_fields) do
-    %w[collection_search pub_date_search db_az_subject_search instructor course topic_display subject_other_display
-       title_variant_display summary_display pub_display]
-  end
-  let(:ignored_fields) do
-    %w[all_search created last_updated format _version_ author_sort callnum_facet_hsim marcbib_xml marcxml mhld_display fund_facet building_facet collection] + copy_fields
-  end
-  let(:pending_fields) { %w[reverse_shelfkey shelfkey preferred_barcode item_display date_cataloged access_facet] }
-  subject(:result) { indexer.map_record(marc_to_folio_with_stubbed_holdings(record)).transform_values { |v| v.sort } }
+  MARC::JSONLReader.new(File.expand_path('input.jsonl', file_fixture_path)).each do |input_marc|
+    context "with #{input_marc['001'].value}" do
+      subject(:result) { indexer.map_record(marc_to_folio_with_stubbed_holdings(input_marc)).transform_values { |v| v.sort } }
+      let(:input_id) { input_marc['001'].value.delete_prefix('a') }
 
-  Dir.glob(File.expand_path('solrmarc_example_docs/*', file_fixture_path)).each do |fixture|
-    context "with #{fixture}" do
-      let(:file) { File.read(fixture) }
-      let(:data) { JSON.parse(file) }
-      let(:solrmarc_doc) { data['doc'] }
       let(:expected_doc) do
-        data['doc'].transform_values { |v| Array(v).map(&:to_s).sort }
+        result = nil
+        File.foreach(File.expand_path('output.jsonl', file_fixture_path)) do |line|
```

Comment:
I'm not sure I understand what's going on here, how we'd add new documents to those jsonl files if we discover other differences between the indexing approaches, or what we'd do when we decide to intentionally diverge from the previous implementation.

Reply:
It's finding the line in the output file with the id that matches the input record. We can add/change/remove any pair of lines from the input and output file if we want to change the tests.